website/snippets/_fusion-upgrade-steps.md

```diff
@@ -0,0 +1,10 @@
+#### Known limitations during upgrade
```

Comment:
stripped this as [Matt's pr](https://github.com/dbt-labs/docs.getdbt.com/pull/7963/files#diff-9a085af40e9903e655ea4a7088b10c54a2aa7d632866cb534d56af443f6750dd) updates the file with the recent upgrade flow